[2021-02-18 22:52:04,939] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:51:00+00:00 [queued]>
[2021-02-18 22:52:04,945] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:51:00+00:00 [queued]>
[2021-02-18 22:52:04,945] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:52:04,945] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:52:04,945] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:52:04,955] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-18T22:51:00+00:00
[2021-02-18 22:52:04,957] {standard_task_runner.py:51} INFO - Started process 19748 to run task
[2021-02-18 22:52:04,963] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-18T22:51:00+00:00', '--job-id', '1168', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmpwe_p4mg4']
[2021-02-18 22:52:04,966] {standard_task_runner.py:76} INFO - Job 1168: Subtask get_tweets_by_user_list
[2021-02-18 22:52:04,995] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:51:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:52:05,023] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-18T22:51:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-18T22:51:00+00:00
[2021-02-18 22:52:05,024] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:52:05,039] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:05,217] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:05,623] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:52:05,631] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:05,803] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:06,308] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:52:06,316] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:06,486] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:07,017] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:52:07,021] {logging_mixin.py:103} INFO - 0:00:01.997246
[2021-02-18 22:52:07,021] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:52:08,993] {logging_mixin.py:103} INFO - 0:00:01.972033
[2021-02-18 22:52:09,014] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:52:09,036] {logging_mixin.py:103} INFO - 0:00:00.042833
[2021-02-18 22:52:09,036] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:52:09,038] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:52:09,041] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210218T225100, start_date=20210218T225204, end_date=20210218T225209
[2021-02-18 22:52:09,061] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:52:09,068] {local_task_job.py:118} INFO - Task exited with return code 0
